### ISSUE TITLE
Dropdown is not responsive

### ISSUE THREAD

**Issue by stephenma064** [jgthms/bulma#909]
### Overview of the problem

This is about the Bulma **CSS framework**

 I'm using Bulma **version** [0.4.4]
 My **browser** is: Google Chrome

### Description

New Dropdown component's dropdown menu is not responsive. So if the dropdown content is too long, the text will keep going past the box in the dropdown menu. Using the responsive helper is-inline-flex makes it stretch out too far. 

![bulma error](https://user-images.githubusercontent.com/5231667/28531519-af648f7e-705c-11e7-9bc4-5462443130bc.PNG)
### Steps to Reproduce

1. Follow the documentation to create dropdown
2. For dropdown-content make the content too long 
3. Attempt to set property to inline-flex

### Expected behavior

Expected to see box fit the contents of the dropdown
### Actual behavior
![bulma actual](https://user-images.githubusercontent.com/5231667/28531647-164d2afc-705d-11e7-8104-69e48480dc4d.PNG)

**Comment by rafaelpimpa** [comment]
This fixes:

```scsss
.dropdown-menu {
    max-width: none; // Fix Bulma v0.4.4
    width: auto;
}
```

**Comment by stephenma064** [comment]
Thank you! That works.
